util/build-img.sh

```diff
@@ -2,18 +2,30 @@
 
 # Build a qcow2 image for loading in qemu
 
-SRC_IMG=$HARVEY/util/img/syslinux-bios
-SRC_SYSLINUX=$SRC_IMG/syslinux
-SRC_MBR=$SRC_IMG/mbr.bin
+# Syslinux should already be installed on the system since it's a
+# dependency of guestfish. We want to use the system syslinux binaries
+# which match the extlinux command below.
+SRC_SYSLINUX=/usr/lib/syslinux/bios
```

Comment:
I'd rather we bring the file into harvey. The reason is that this path is not at at all standard across distros. Would that be ok with you? These files change rarely if ever.

Reply:
I'm ok with that. Any idea where the existing syslinux files were obtained from? I need the `extlinux` binary that matches `mbr` and other syslinux files already in harvey. Otherwise, I could just updated all syslinux files to the latest version.

Reply:
Might as well update all with the latest

Reply:
I thought guestfish searches for `extlinux` in `$PATH`. I was wrong. It creates a VM image from the host system and runs everything from within qemu. So, modifying `$PATH` to include harvey's `extlinux` will not prevent guestfish from picking up the system `extlinux`. I don't see any easy workaround for this.

We can run `extlinux` externally after mounting the filesystem using `qemu-nbd` but that'd require root access and kind of defeats the purpose of using guestfish. Maybe we can just check for some common places linux distros keep the syslinux `*.c32` files?

Ultimately we want to provide downloadable boot images to users who want to try out harvey, so this script will be run by only few people.